deploy/spoke/clusterrole.yaml

```diff
@@ -15,4 +15,6 @@ rules:
 - apiGroups: ["work.open-cluster-management.io"]
   resources: ["appliedmanifestworks/finalizers"]
   verbs: ["update"]
-  
\ No newline at end of file
+- apiGroups: ["authorization.k8s.io"]
```

Comment:
nit: you may want to add this in operator at first.

Reply:
yes, I  added it https://github.com/open-cluster-management-io/registration-operator/pull/273